src/game_map.cpp

```diff
@@ -1329,17 +1329,13 @@ void Game_Map::UpdateParallax() {
 		return;
 
 	if (map_info.parallax_horz && map_info.parallax_horz_auto) {
-		parallax_x +=
-			map_info.parallax_horz_speed > 0 ? 1 << (map_info.parallax_horz_speed - 1) :
-			map_info.parallax_horz_speed < 0 ? -1 << (-map_info.parallax_horz_speed - 1) :
-			0;
+		int acc = 1 << (std::abs(map_info.parallax_horz_speed) - 1);
```

Comment:
I guess `parallax_horz` and/or `parallax_horz_auto` was always false and so a speed of `0` does not exist.

Well, i will just add the check back in, in the condition above, as it does not hurt to have.